test/e2e/network/netpol/network_policy.go

```diff
@@ -1088,6 +1088,44 @@ var _ = common.SIGDescribe("Netpol [LinuxOnly]", func() {
 			reachability.ExpectAllIngress(NewPodString(nsX, "a"), false)
 			ValidateOrFail(k8s, model, &TestCase{ToPort: 81, Protocol: v1.ProtocolTCP, Reachability: reachability})
 		})
+
+		ginkgo.It("should enforce policy to allow traffic based on NamespaceSelector with MatchLabels using default ns label [Feature:NetworkPolicy][Feature:NamespaceDefaultLabelName]", func() {
```

Comment:
@mattfenwick I am not sure that whether we need this label `[Feature:NamespaceDefaultLabelName]` ?

Reply:
yeah good question @rpkatz what do you think ?

Reply:
ah whoops pinged wrong handle ... @rikatz there we go